source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +22,7 @@ ReferenceCounter: class {
 			}
 		}
 	}
+	count ::= this _count
```

Comment:
> this fails the ReferenceCounter test

That's bad. Setters that call their corresponding getter should be banned at once.